storage/filesystem/storage_test.go

```diff
@@ -14,31 +15,29 @@ func Test(t *testing.T) { TestingT(t) }
 
 type StorageSuite struct {
 	test.BaseStorageSuite
+	dir string
 }
 
 var _ = Suite(&StorageSuite{})
 
 func (s *StorageSuite) SetUpTest(c *C) {
-	storage, err := NewStorage(osfs.New(c.MkDir()))
+	s.dir = c.MkDir()
+	storage, err := NewStorage(osfs.New(s.dir))
 	c.Assert(err, IsNil)
 
 	s.BaseStorageSuite = test.NewBaseStorageSuite(storage)
 }
 
-func (s *StorageSuite) TestNewStorage(c *C) {
+func (s *StorageSuite) TestFilesystem(c *C) {
 	fs := memfs.New()
 	storage, err := NewStorage(fs)
 	c.Assert(err, IsNil)
-	c.Assert(storage, NotNil)
 
-	_, err = fs.Stat("refs/tags")
```

Comment:
Its only purpose was checking that refs/tags was created before any action. This is no longer the case, so it's enough if we check the other cases.